db.json

```diff
@@ -2351,7 +2419,8 @@
     "source": "iana"
   },
   "application/vnd.debian.binary-package": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["deb","ddeb","udeb"]
```

Comment:
I see deb and udeb listed in the type registration https://www.iana.org/assignments/media-types/application/vnd.debian.binary-package , but where does ddeb come from?